README.md

```diff
@@ -32,6 +32,8 @@ kas-docker build meta-rte/kas.yml
 
 ## Flash
 
+### etcher
```

Comment:
@WiktorG351 under `## Flash` header please mention that we have two known ways to flash image, etcher and bmaptool. It i s a good practice to add at least one sentence for any given section so we not end up wit

# Header 1

## Header 2

### Header 3

#### Header 4

##### Header 5

Lorum ipsum

Reply:
fixed in https://github.com/3mdeb/meta-rte/blob/734d074babc8652f7e6ba7f7ada1e5a296b60f0b/README.md?plain=1#L35